src/index.html

```diff
@@ -17,16 +17,22 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="search-box search-box--big">
+      <input
+        class="search-txt search-txt--big"
+        type="text"
+        data-qa="big"
+        placeholder="Try “Los Angeles“"
+      />
```

Comment:
Add empty lines between multiline sibling blocks of HTML to add some 'air' and simplify reading. There should be an empty line between the closing `</div>` on line 27 and the opening `<div>` on line 29.